### PR TITLE
Update SendGrid Sample due to library changes

### PR DESCRIPTION
**Pull request by averikitsch**

### CODE REVIEW

File: appengine-java8/sendgrid/pom.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!--
-Copyright 2015 Google Inc.
+Copyright 2018 Google Inc.
```
Comment (dzlier-gcp) [inline appengine-java8/sendgrid/pom.xml]: If updating the copyright, please also change `Inc.` to `LLC` (in other files as well)

Comment (averikitsch) [reply appengine-java8/sendgrid/pom.xml]: done.

Comment (averikitsch) [reply appengine-java8/sendgrid/pom.xml]: Kokoro is set to look for Google Inc. not Google LLC.

**Comment by dzlier-gcp** [comment]
Ah, I see that broke a checkstyle check - this should be fixed by updating shared-configuration to 1.0.10.

**Pull request merged**